### DISCUSSION TITLE
page status icons and example

### DISCUSSION THREAD

**Discussion by PlasmaHH** [squidfunk/mkdocs-material#7246] [General]
I had a look at the [page status example](https://mkdocs-material.github.io/examples/page-status/) and wondered a couple of things:

Where is the code for it? clicking on the "view source" gives me a 404

I noticed in the delivered css there is a mention of an encryption icon and status which is not documented. Is it an official feature that just wasn't documented?

Is it possible to easier ( maybe in the yaml ) create more such status icons using the bundled ones? The css seems to be always the same, just name and svg data change...

**Comment by alexvoss** [comment]
Thanks for letting us know. I will have a look to see why the page source link is broken. In the meantime, the code can be found at https://github.com/mkdocs-material/examples/tree/master/examples/page-status

**Reply by alexvoss** [reply]
So, the source/edit urls are produced by a little helper script in the [examples source](https://github.com/mkdocs-material/examples/blob/master/examples/__init__.py), which needs to be fixed, I think. All the source/edit links for the examples are broken at the moment. Tagging @squidfunk in case he already knows what needs to change.

**Reply by squidfunk** [reply]
Thanks for mentioning. So the links from the documentation all work, but the source/edit links have problems, yes. I'll check it out.

> I noticed in the delivered css there is a mention of an encryption icon and status which is not documented. Is it an official feature that just wasn't documented?

It's still undocumented better support for the encryptcontent plugin. I'm still on the fence whether we should support it natively.

> Is it possible to easier ( maybe in the yaml ) create more such status icons using the bundled ones?

No, currently not possible. Please follow the guidance on adding custom status icons that we already gave in other discussions here.

**Reply by squidfunk** [reply]
Fixed, see https://mkdocs-material.github.io/examples/page-status/

**Reply by alexvoss** [reply]
Thanks for fixing this so swiftly!
